app/page.tsx

```diff
@@ -16,9 +18,32 @@ export default async function Home() {
       },
       feeStatus: true,
       wideningParticipation: true,
-      nextAction: true
+      nextAction: true,
+      reviewer: {
+        select: {
+          loginId: true
+        }
+      }
     }
   })
 
-  return <ApplicationTable applications={applications} />
+  const reviewerIds = (
+    await prisma.user.findMany({
+      select: {
+        loginId: true
+      },
+      where: {
+        role: Role.REVIEWER
+      },
+      orderBy: {
+        loginId: 'asc'
+      }
+    })
+  ).map((user) => user.loginId)
+
+  return (
+    <>
+      <ApplicationTable applications={applications} reviewerIds={reviewerIds} />
+    </>
+  )
```

Comment:
```suggestion
  return (
    <ApplicationTable applications={applications} reviewerIds={reviewerIds} />
  )
```

Reply:
resolved in 3993a3331e005e1ae1fa3b79a364ca8ee3c2f24f